blockbase/functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+if ( ! function_exists( 'blockbase_support' ) ) :
+	function blockbase_support()  {
```

Comment:
Many of these settings should either be activated by default for block-based themes, or available as `theme.json` options:

```
add_theme_support( 'align-wide' );
add_theme_support( 'wp-block-styles' );
add_theme_support( 'responsive-embeds' );
add_theme_support( 'experimental-link-color' );
```

Reply:
Is there an issue for this in Gutenberg?

Reply:
I just opened https://github.com/WordPress/gutenberg/issues/26901 to discuss.